src/components/Form/Form.tsx

```diff
@@ -26,6 +28,12 @@ export interface FormProps extends UIComponentProps, ChildrenComponentProps {
    * @param {object} data - All props.
    */
   onSubmit?: ComponentEventHandler<FormProps>
+
+  /**
+   * Rtl attributes function if overridden by the user.
+   * @default formRtlAttributes
+   */
+  rtlAttributes?: RtlFunc
```

Comment:
it seems like it would be more expressive if we'll rename type to `RtlAttributesProvider`, what do you think?

Reply:
Agreed, let me rename it.